docs/reference/commandline/service_create.md

```diff
@@ -779,6 +780,24 @@ $ docker service create --name dns-cache -p 53:53/tcp -p 53:53/udp dns-cache
 $ docker service create --name dns-cache -p 53:53/udp dns-cache
 ```
 
+### Provide credential specs for managed service accounts (Windows only)
+
+This option is only used for services using Windows containers. The
+`--credential-spec` must be in the format `file://filename.json` or
+`registry://value-name`.
+
+When using the `file://` format, the referenced file must be present
+must be present in the `CredentialSpecs` subdirectory in the docker
+data directory, which defaults to `C:\ProgramData\Docker\` on Windows.
+For example, specifying `file://spec.json` loads
+`C:\ProgramData\Docker\CredentialSpecs\spec.json`.
+
+When using the `registry://keyname` format, the credential spec is read
```

Comment:
Remove `keyname` here since this only needs to state the prefix (and, technically, it is a value not a key).